### PR TITLE
Updated EcsParameters in events.py

### PR DESCRIPTION
**Pull request by pfoote**
Updated EcsParameters in events.py to include Group, LaunchType, NetworkConfiguration and PlatformVersion.

### CODE REVIEW

**Comment by markpeek** [comment]
These additional properties are not implemented in [CloudFormation yet](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-events-rule-ecsparameters.html).

**Comment by cmmeyer** [comment]
FYI -- I've created an internal ticket to track this on the AWS CloudFormation side.

**Comment by helloangely** [comment]
CloudFormation just released support for this! On August 29th

Docs: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-events-rule-ecsparameters.html

Release notes: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/ReleaseHistory.html

File: troposphere/events.py [hunk 1]
```diff
@@ -4,12 +4,17 @@
...
+from ecs import AwsvpcConfiguration
 
 
 class EcsParameters(AWSProperty):
     props = {
-        "TaskCount": (int, False),
-        "TaskDefinitionArn": (basestring, True),
+        'Group': (basestring, False),
+        'LaunchType': (basestring, False),
+        'NetworkConfiguration': (AwsvpcConfiguration, False),
```
Comment (helloangely) [inline troposphere/events.py]: @pfoote I got this to work using type `ecs.NetworkConfiguration` as well

**Comment by markpeek** [comment]
Sorry, missed the comments to this issue and committed this change via 16666819d733b1f1bc8ea494199f53bfe8531686. Released in version 2.5.2.

**Comment by helloangely** [comment]
Very excited to use, thank you!

**Pull request closed without merging**